main/src/ca/uwaterloo/flix/language/ast/ParsedAst.scala

```diff
@@ -1051,7 +1049,7 @@ object ParsedAst {
       * @param exp3  the else expression.
       * @param sp2   the position of the last character in the expression.
       */
-    case class ReifyEff(sp1: SourcePosition, exp1: ParsedAst.Expression, ident: Name.Ident, exp2: ParsedAst.Expression, exp3: ParsedAst.Expression, sp2: SourcePosition) extends ParsedAst.Expression
+    case class ReifyPur(sp1: SourcePosition, exp1: ParsedAst.Expression, ident: Name.Ident, exp2: ParsedAst.Expression, exp3: ParsedAst.Expression, sp2: SourcePosition) extends ParsedAst.Expression
```

Comment:
I think our naming rule should be that we abbreviate variables, but not types/constructors. So this should be `ReifyPurity` (even though it was ReifyEff before).